### PR TITLE
[hue] improved readme

### PR DESCRIPTION
**Pull request by kaikreuzer**
Migrated from https://github.com/eclipse/smarthome/pull/6829

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: addons/binding/org.openhab.binding.hue/README.md [hunk 1]
```diff
@@ -224,14 +225,13 @@ And there is one Hue Motion Sensor (represented by three devices) and a Hue Dimm
...
-Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
-    0210 bulb1 [ lightId="1" ]
-    0220 bulb2 [ lightId="2" fadetime=300 ]
-    0106 light-level-sensor [ sensorId="3" ]
-    0107 motion-sensor [ sensorId="4" ]
-    0302 temperature-sensor [ sensorId="5" ]
-    0820 dimmer-switch [ sensorId="6" ]
-}
+Bridge hue:bridge:1 "Hue Bridge" [ ipAddress="192.168.0.64" ] {
+    0210 bulb1          "Lamp 1"        @ "Kitchen" [ lightId="1" ]
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.hue/README.md]: The indentation looks weird, but it does not matter.

File: addons/binding/org.openhab.binding.hue/README.md [hunk 2]
```diff
@@ -224,14 +225,13 @@ And there is one Hue Motion Sensor (represented by three devices) and a Hue Dimm
...
 
 ```
-Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
-    0210 bulb1 [ lightId="1" ]
-    0220 bulb2 [ lightId="2" fadetime=300 ]
-    0106 light-level-sensor [ sensorId="3" ]
-    0107 motion-sensor [ sensorId="4" ]
-    0302 temperature-sensor [ sensorId="5" ]
-    0820 dimmer-switch [ sensorId="6" ]
-}
```
Comment (wborn) [inline addons/binding/org.openhab.binding.hue/README.md]: It does matter that the `}` is now missing. :wink:

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.hue/README.md]: Oh, yes, I missed that. Thanks.

Comment (kaikreuzer) [reply addons/binding/org.openhab.binding.hue/README.md]: fixed!

**Pull request merged**